sme_finance_application_schema/test.py

```diff
@@ -1,10 +1,155 @@
+    'company_credit_rating':'ok',
+    'personal_credit_ratings':'ok',
+    'directors_houses':100000,
+    'exports':True,
+    'total_value_of_unsatisfied_ccjs':1000,
+    'profitability':50,
+    'institutional_revenue':50,
+    'up_to_date_accounts':True,
+    'date_of_first_filed_accounts': '2015-01-23T00:00:00+00:00',
+    'count_of_invoiced_customers': 100,
+    'outstanding_invoices': 1000,
+}
+
+SME_CONTACT_V3 = {
+    'applicant_title': 'Mr',
+    'applicant_first_name': 'Dave',
+    'applicant_surname': 'dd',
+    'telephone': '+447445387241',
+    'sme_name': 'ddsaasd',
+    'email': 'nestor.arocha@fundingoptions.com',
+    'company_number': '123456',
+    'address_line_1': '30 Great Guildford Street',
+    'address_line_2': 'Derp',
+    'postcode': 'SE1 0HS',
+    'city': 'London',
+    'county': 'London',
+}
+
+FINANCE_APPLICATION_V3 = {
+    'applicant': {
+        'title': 'Mr',
+        'first_name': 'Dave',
+        'surname': 'dd',
+        'telephone':'+447445387241',
+        'email': 'nestor.arocha@fundingoptions.com',
+        'addresses': [{
+            'address': {
+                'building_number_and_street_name': '30 Great Guildford Street',
+                'post_town': 'London',
+                'postcode': 'SE1 0HS',
+            }
+        }]
+    },
+    'finance_need': {
+        'finance_term_length': 30,
+        'purpose': 'stock',
+        'date_finance_requested': '2017-01-23T00:00:00+00:00',
+        'date_finance_required': '2018-01-23T00:00:00+00:00',
+        'requested_amount': 3000,
+        'finance_type_requested':'term_loan',
+        'guarantor_available':True,
+    },
+    'requesting_entity': {
+        'name': 'ddsaasd',
+        "company_number": "123456",
+        'legal_status': 'limited_company',
+        'months_revenue':0,
+        'trade_credit':0,
+        'revenue':2000,
+        'sic_code':'A',
+        'card_revenue':3000,
+        'financial_forecast':True,
+        'revenue_growth':50,
+        'purchase_orders':50,
+        'registered_brand':True,
+        'stock_imports':50,
+        'customers':100,
+        'business_plan':True,
+        'stock_ready':50,
+        'business_premises':50000,
+        'region':'UKZ',
+        'intellectual_property':True,
+        'business_assets':30000,
+        'accounting_software':'xero',
+        'overseas_revenue':50,
+        'online_revenue':50,
+        'company_credit_rating':'ok',
+        'exports':True,
+        'profitability':50,
+        'institutional_revenue':50,
+        'up_to_date_accounts':True,
+        'registration_date': '2012-01-23T00:00:00+00:00',
+        'employees':50,
+        'outstanding_invoices': 1000,
+        'count_of_invoiced_customers':100,
+        'total_value_of_unsatisfied_ccjs':1000
+    },
+    'actors': []
+}
+
+# The following are fields that do not appear in the objects when they are translated from finance_application_v3
+
+UNTRANSLATED_SME_V5_FIELDS = [
+    'directors_pensions', #Aggregate
```

Comment:
we translate this
https://github.com/FundingOptions/sme-finance-application-schema/blob/19d8f49b7884d051bf4c64bd1b21b5accb050356/sme_finance_application_schema/translations.py#L51-L53